tests/e2e/specs/drawing/kml.spec.js

```diff
@@ -15,7 +15,7 @@ describe('Drawing saving KML', () => {
                     cy.goToDrawing(isMobileViewport || isTabletViewport)
                 })
 
-                it('saves a KML on draw end', () => {
+                it.only('saves a KML on draw end', () => {
```

Comment:
don't forget to remove your `.only` calls after fixing the test 😉

Reply:
Dang! :laughing:

Reply:
Done. I allowed myself to amend the last commit.

The only changes are the removal of the `.only` and I updated the clicks by taking the values from the `drawGeoms` custom task.